packages/tocco-ui/src/EditableValue/typeEditors/TimeEdit.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import DurationEdit from './DurationEdit'
+import {calculateMilliseconds} from '../utils'
+
+const TimeEdit = props => {
+  const propHours = (props.value === null || props.value === undefined) ? 0 : props.value.hoursOfDay
```

Comment:
Maybe move (props.value === null || props.value === undefined) into a var. Or create a helper function "isNullOrUndefined" i think this boilerplate code makes the code hard to read